src/FsLex.Core/fslexpars.fsy

```diff
@@ -26,9 +26,12 @@ spec:
     { TopCode=$1;Macros=$2;Rules=$4;BottomCode=$5 }
 }
 
+ident:
```

Comment:
the parser for fslex files itself had to be updated to adjust to the new AST shapes, supplying ranges where the types required it, or returning a record instead of a bare tuple.